### DISCUSSION TITLE
Paid for licence... not received key.

### DISCUSSION THREAD

**Discussion by MarkBurvs** [airsdk/Adobe-Runtime-Support#798] [General]
Has anyone else had trouble with the licence purchasing process? We paid for our Harman AIR license ten days ago but have not received any confirmation (beyond the paypal confirmation) or licence key yet. I emailed the adobe.support@harman.com address to chase, but also heard nothing.

**Comment by ajwfrost** [comment]
Hi - sorry, I just got forwarded your email from our billing team - I'll check on this one and will also check with the folk who are meant to be monitoring that inbox..

**Reply by ajwfrost** [reply]
FYI: paypal-based payments should be automatically dispatching the licenses, except for US-based clients where we may need to have sales tax added depending on (a) where you are (state/county I think) and (b) what our tax filing status is for that state. Hence a little bit of a delay can happen sometimes but it should be no more than a day or two!

thanks

**Reply by MarkBurvs** [reply]
Ah okay thanks Andrew. We're in the UK though so something maybe went a bit wrong. No harm done as I was off last week anyway, but would be good to get the key now so I can get the update submitted.
